### PR TITLE
docs: add README first version

### PR DESCRIPTION
**Pull request by aleortega**
Add first version of the README explaining how to consume this service as a point of start.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,62 @@
-# template-server
+# Textures Helper
 
-## Architecture
+Textures helper compress textures using DXT1 mipmapped format.
```
Comment (marianogoldman) [inline README.md:3]:
```suggestion
Textures helper compresses textures using DXT1 mipmapped format.
```

File: README.md [hunk 2]
```diff
@@ -1,50 +1,62 @@
...
 
-Extension of "ports and adapters architecture", also known as "hexagonal architecture".
+## Converting an image
 
-With this architecture, code is organized into several layers: logic, controllers, adapters, and components (ports).
+This service exposes and endpoint in charge of compressing textures which expects two inputs `size` as a path parameter and `asset` as a query parameter. Below can be found the call structure to this endpoint:
```
Comment (marianogoldman) [inline README.md:7]:
```suggestion
This service exposes an endpoint for compressing textures which expects two inputs: `size` as a path parameter and `asset` as a query parameter. The call structure for this endpoint is as follows:
```

File: README.md [hunk 3]
```diff
@@ -1,50 +1,62 @@
...
 
-## Application lifecycle
+```
+GET <HOST>/content/dxt/:LENGTH?asset=<ASSET-URL>
+```
+
+* **length** - must be an integer of a power of two that can go from 128 until 2048
```
Comment (marianogoldman) [inline README.md:13]:
```suggestion
* **length** - must be an integer and power of two that can go from 128 until 2048
```

File: README.md [hunk 4]
```diff
@@ -1,50 +1,62 @@
...
+* **asset** - must be a HTTP url aiming to the asset to be converted by the service
```
Comment (marianogoldman) [inline README.md:14]:
```suggestion
* **asset** - must be a HTTP url referencing the asset to be converted by the service
```

File: README.md [hunk 5]
```diff
@@ -1,50 +1,62 @@
...
+  "conversionRatio":
+    {
+      "height": number,
+      "width": number
+     }
+}
+```
 
-## Namespaces
+* **asset** - contains the URL of the resized and compressed asset. This URL will aim to the CDN configured for this service.
```
Comment (marianogoldman) [inline README.md:29]:
```suggestion
* **asset** - contains the URL of the resized and compressed asset. This URL will point to the CDN configured for this service.
```

File: README.md [hunk 6]
```diff
@@ -1,50 +1,62 @@
...
+* **conversionRatio** - contains the resize ratio relating old asset size with the new one specified when calling the endpoint
+  * **height** - newHeight / oldHeight (_fixed to four decimals_)
```
Comment (marianogoldman) [inline README.md:31]: Being a `number`, does this need to be rounded? In general rounding applies for string types...

**Review by marianogoldman — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (aleortega) [reply README.md:31]: I'll check this, if confirmed that this is not needed I'll update the code and this part of the documentation in a new PR.

Thank you!

**Review by marianogoldman — APPROVED** [review]

**Pull request merged**